plugins/modules/jira.py

```diff
@@ -480,6 +501,8 @@ class JIRA(StateModuleHelper):
             username=dict(type='str'),
             password=dict(type='str', no_log=True),
             token=dict(type='str', no_log=True),
+            client_cert=dict(type='path'),
+            client_key=dict(type='path'),
             project=dict(type='str', ),
             summary=dict(type='str', ),
             description=dict(type='str', ),
```

Comment:
Further below you'll find fields `mutually_exclusive`, `required_together`, `required_one_of`, which you'll also need to update accordingly. (Whatever the actual requirements are right now.)